src/explorer/webview.ts

```diff
@@ -1,66 +1,75 @@
-// /********************************************************
-// *                                                       *
-// *   Copyright (C) Microsoft. All rights reserved.       *
-// *                                                       *
-// ********************************************************/
+/*!
```

Comment:
This legacy DOM construction/manipulation strategy is at its scaling limit and is not time/cost-effective. Recommend avoiding further investment (such as refactoring). This includes time spent reviewing. If the intent is to add major features here, then we should discuss strategy.

Reply:
I couldn't agree more. Not super worth it in this PR.

---
In reply to: [397721726](https://github.com/microsoft/sarif-vscode-extension/pull/211#discussion_r397721726) [](ancestors = 397721726)